espwrap/adaptors/sendgrid.py

```diff
@@ -139,4 +139,8 @@ def send(self):
                 if self.important:
                     msg.set_headers({'Priority': 'Urgent', 'Importance': 'high'})
 
+                if self.attachments:
+                    for file_name, file_path_or_string in self.attachments.iteritems():
+                        msg.add_attachment(file_name, file_path_or_string)
```

Comment:
the sendgrid api method 'add_attachment' looks like this btw:

    def add_attachment(self, name, file_):
        if sys.version_info < (3, 0) and isinstance(name, unicode):
            name = name.encode('utf-8')
        if isinstance(file_, str):  # filepath
            with open(file_, 'rb') as f:
                self.files[name] = f.read()
        elif hasattr(file_, 'read'):
            self.files[name] = file_.read()